python/pyspark/status.py

```diff
@@ -83,6 +85,8 @@ def getJobInfo(self, jobId):
         job = self._jtracker.getJobInfo(jobId)
         if job is not None:
             return SparkJobInfo(jobId, job.stageIds(), str(job.status()))
+        else:
```

Comment:
Yeah, I know...  Hmmm, this is from something else, not sure how it got mixed in.  Let me remove it.